packages/kamino-sdk/src/utils/types.ts

```diff
@@ -158,6 +158,11 @@ export interface GenericPoolInfo {
   positions: number;
 }
 
+export interface GenericPositionRangeInfo {
+  estimatedApy: number;
```

Comment:
Probably better to make it a Decimal instead

Reply:
agree